metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and configures haproxy'
-long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
+
```

Comment:
Can you remove this extra line as well please? (It's a known thing in cookstyle which is set to be resolved in the next release)
```suggestion
```